ProcessLib/Output/Output.h

```diff
@@ -87,32 +87,32 @@ class Output
                         double const t) const;
 
 private:
-    std::string const _output_directory;
-    std::string const _output_file_prefix;
-    std::string const _output_file_suffix;
+    std::string const output_directory_;
+    std::string const output_file_prefix_;
+    std::string const output_file_suffix_;
 
     //! Enables or disables zlib-compression of the output files.
-    bool const _output_file_compression;
+    bool const output_file_compression_;
 
     //! Chooses vtk's data mode for output following the enumeration given in
     /// the vtkXMLWriter: {Ascii, Binary, Appended}.  See vtkXMLWriter
     /// documentation http://www.vtk.org/doc/nightly/html/classvtkXMLWriter.html
-    int const _output_file_data_mode;
-    bool const _output_nonlinear_iteration_results;
+    int const output_file_data_mode_;
+    bool const output_nonlinear_iteration_results_;
 
     //! Describes after which timesteps to write output.
-    std::vector<PairRepeatEachSteps> _repeats_each_steps;
+    std::vector<PairRepeatEachSteps> repeats_each_steps_;
 
     //! Given times that steps have to reach.
-    std::vector<double> _fixed_output_times;
+    std::vector<double> fixed_output_times_;
 
-    std::multimap<Process const*, MeshLib::IO::PVDFile> _process_to_pvd_file;
+    std::multimap<Process const*, MeshLib::IO::PVDFile> process_to_pvd_file_;
 
     /**
      * Get the address of a PVDFile from corresponding to the given process.
-     * @param process    Process.
+     * @param process Process.
      * @param process_id Process ID.
-     * @param mesh_name_for_output mesh name for the output.
+     * @param mesh_name_for_output Mesh file name for output
```

Comment:
Still, there are unrelated changes in the comments.

Reply:
> Still, there are unrelated changes in the comments.

keen eye! Thanks.
Forget to run grep for checking after rebase. After ran grep '#ifdef' and '#if defined'. The follwing macro definitions were corrected
```
Applications/DataExplorer/mainwindow.cpp:#ifdef WIN32_
GeoLib/Polygon.cpp:#if _GNUC___ <= 4 && (_GNUC_MINOR___ < 9)
GeoLib/Polygon.h:#if _GNUC___ <= 4 && (_GNUC_MINOR___ < 9)
```


Still, there are unrelated changes in the comments.
MathLib::Point3d getBulkElementPoint(MeshLib::Quad const& quad,

Reply:
> Still, there are unrelated changes in the comments.

These were changed for correcting doxygen error from master. I saw that the doxygen errors  in master have been corrected now. I just removed the changes in documentation including another in
```
MathLib::Point3d getBulkElementPoint(MeshLib::Quad const& quad,
```